source/includes/_snapshots.md

```diff
@@ -1 +1,126 @@
 # Snapshots
+
+## The Snapshot Resource
+
+### Attributes:
+
+
+Attribute | Type | Description
+--------- | ------- | -------
+`id` | string | A unique identifier of this snapshot.
+`created_at` | datetime | The time the snapshot was uploaded to or created on the manager.
+`status` | string | Status of this snapshot - created/failed/uploading/uploaded.
+`error` | string | Message of an error if snapshot creation failed.
+
+## List Snapshots
+`GET /api/v2/snapshots`
+
+Lists all snapshots
+
+> Request Example
+
+```shell
+$ curl -XGET http://localhost/api/v2/snapshots
+```
+
+> Response Example
+
+```json
+{
+    "items": [
+        {
+            "created_at": "2015-12-04 13:34:45.080009",
+            "error": "",
+            "id": "snapshot1",
+            "status": "created"
+        },
+        {
+            "created_at": "2015-12-04 13:35:04.972249",
+            "error": "",
+            "id": "snapshot2",
+            "status": "created"
+        }
+    ],
+    "metadata": {
+        "pagination": {
+            "offset": 0,
+            "size": 10000,
+            "total": 2
+        }
+    }
+}
+```
+
+### Response
+
+Field | Type | Description
+--------- | ------- | -------
+`items` | list | A list of `Snapshot` resources.
```

Comment:
You can have a link to the `Snapshot` resource.. use [Snapshot](#the-snapshot-resource)